straxen/plugins/event_shadow_test.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
```

Comment:
Why do we need this plugin?

Reply:
I should have deleted this file. This is a very early version.